### ISSUE TITLE
External NMEA GNSS Compass GPS1_TYPE param incorrectly referenced

### ISSUE THREAD

**Issue by lashVN** [ArduPilot/ardupilot_wiki#6206]
Referencing [this](https://ardupilot.org/plane/docs/common-gps-for-yaw.html#nmea) page, there is a link to GPS1_TYPE, which as far as I can tell doesn't exist in any recent firmware. While I'm not familiar with the code, by my eye it should probably reference [this ](https://ardupilot.org/plane/docs/parameters.html#gps-type-1st-gps-type) param?

**Comment by Hwurzburg** [comment]
no, GPS_TYPE was changed to GPS1_TYPE in master(4.6), so wiki pages reflect that because only master params are hyperlinked...why its still shown,along with all the other old param names on the wiki 4.6 params page needs to be investigated however@peterbarker
